plugins/data.uribl.js

```diff
@@ -106,11 +109,13 @@ exports.do_lookups = function (connection, next, hosts, type) {
                 results.add(plugin, {skip: type + ' unsupported for ' + zone });
                 continue;
             }
-            // Convert in-addr.arpa into bare IPv4 lookup
+            // Convert in-addr.arpa into bare IPv4/v6 lookup
             var arpa = host.split(/\./).reverse();
-            if (arpa.shift() === 'arpa' && arpa.shift() === 'in-addr') {
-                if (arpa.length < 4) continue; // Only full IP addresses
-                host = arpa.join('.');
+            if (arpa.shift() === 'arpa'){
+                if ( ['in-addr','ip6'].indexOf(arpa.shift()) !== -1) {
+                    if (arpa.length < 4) continue; // Only full IP addresses
```

Comment:
This code is looking for 4.3.2.1.in-addr.arpa and converting that into 4.3.2.1 instead.

You've added a check for ip6.arpa, but that's not going to work properly being as there's no length check when it is IPv6 e.g. arpa.length < 4 is fine for IPv6, but not IPv6, so it's slightly wrong.